synapse/replication/tcp/resource.py

```diff
@@ -205,6 +205,13 @@ async def _run_notifier_loop(self) -> None:
                             # send, so we send a `POSITION` to inform other
                             # workers of the updated position.
 
+                            # We skip this for the `caches` stream as a) it
+                            # generates a lot of traffic as every worker would
+                            # echo each write, and b) nothing cares if a given
+                            # worker's caches stream position lags.
```

Comment:
Does this mean that workers don't need to track other workers cache stream positions?

Reply:
The difference is nothing uses the global "persisted upto position" for the caches stream, as everything that uses the caches streams only care about each streams position, not any sort of "combined" position. The reason we care about that for other streams is there is some code that still thinks that all streams can be considered as a single integer (e.g. federation and application sending code).

Reply:
> global "persisted upto position" for the caches stream

Just to refresh my memory -- this is the minimum persisted position across all workers, correct?

> everything that uses the caches streams only care about each streams position

Because the caches stream gets blasted to everyone and they just clear their caches but we don't really care about tracking it? Do we even care about each stream's position in it?